### DISCUSSION TITLE
hardware issue

### DISCUSSION THREAD

**Discussion by kyoeun-space** [bitcraze/discussions#877] [Q&A]
![image](https://github.com/bitcraze/discussions/assets/122524815/8277c7d7-3953-4e51-9416-718e8f1687d5)

Hello!
m1 led of my crazyflie 2.1 blinks red light almost 5 times a sec.
the picture is what i get in the console.

is this a problem of hardware or matter of sensing?
If it's a sensing problem, how can I fix it?

Thank you.

**Comment by tobbeanton** [comment]
There is most likely something wrong with the flow deck. Might be a contact problem. Try removing it to see that the CF2.1 boots normally. Reconnect it and it might start working.

**Reply by kyoeun-space** [reply]
Is it right that flow deck can be connected to bolt 1.1? Does flow deck use I2C or SPI?
now I'll try it again, thanks!

**Reply by tobbeanton** [reply]
Yes it can but be cautious! The flow deck is not tracking perfectly as you probably already know. Make sure to tune the PIDs and set the baseThrust parameter with a decent value.

The [flow deck uses both I2C and SPI](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck-pin-allocation).

**Reply by kyoeun-space** [reply]
it's cool. thank you so much!
